tests/dynamic_data/publications/layer_raster_contrast/contrast_test.py

```diff
@@ -118,6 +118,21 @@
             },
         },
     },
+    'tif_float_alpha_style': {
```

Comment:
- move to file_input
- rename file_input to edge_cases
- tif_float_alpha_style -> styled_raster_issue_681